build

```diff
@@ -7,6 +7,8 @@ import glob
 import fnmatch
 import multiprocessing as mp
 
+import argparse
+
```

Comment:
The thing that came to my mind: is the `argparse` part of the standard Python library, because if it isn't, it would introduce unnecessary dependency for users trying to build MAGE.

Reply:
I didn't have to install anything - and also it is library on Python docs so I think it is
https://docs.python.org/3/library/argparse.html

Reply:
I double-checked - it is

Reply:
Awesome 😄 put it on approve 🛩️